src/host/ut_host/SelectionTests.cpp

```diff
@@ -343,68 +343,6 @@ class SelectionTests
         VERIFY_ARE_EQUAL(srOriginal.Left + sDeltaLeft, srSelection.Left);
         VERIFY_ARE_EQUAL(srOriginal.Right + sDeltaRight, srSelection.Right);
     }
-
-    TEST_METHOD(TestBisectSelection)

```

Comment:
Hmm. For some tests, it may be good practice to leave a comment on the PR explaining why it's not necessary any more. :)

Reply:
"Bisect" in this case stands for "wide characters that are stored in the first/last cell of a row", which we don't support anymore, even with `CHAR_INFO`s. So this test was removed.

Reply:
> As in, this PR breaks that functionality? Or we officially stopped supporting it a while back?
> Why is it ok that we're dropping support for that? I seem to be missing some context here haha.

Both. Kinda. This PR breaks it harder than we already broke it in the past.
I'm not sure when, but at some point, we stopped supporting storage of "half a wide glyph" in the first/last column of the console ("bisecting rows"). Instead it was supposed to either store a whole wide glyph or none at all. With any of the public console APIs it shouldn't have been possible to bring conhost into a state where it stores such bisecting glyphs in a long while.
But we still kind of supported it internally, for instance when you rapidly resized the console, you can bring it into a state, where the reflow algorithm puts glyphs into the wrong columns. This PR puts the last nail into the coffin and ensures that no one can mess with our TextBuffer anymore, not even ourselves, so this test can't possibly work, just like you already couldn't do that via any of the public APIs. To be entirely honest, I'm not even 100% sure why `TestBisectSelection` still existed in the first place, and our git history didn't bring up any clues for that either.

BTW a bug caused by bisecting glyphs: https://github.com/microsoft/terminal/pull/14275
I'm sure there's tons more of that lingering around in our code base.